docs/general/attributes.md

```diff
@@ -209,6 +142,48 @@ if they do not cause breaking changes to HTTP semantic conventions.
 | `ipv6` | IPv6 |
 <!-- endsemconv -->
 
+### `network.peer.*` and `network.local.*` attributes
+
+These attributes identify network peers that are directly connected to each other.
+
+For IP-based communication, `network.peer.address` and `network.local.address` should be IP addresses, Unix domain names, or other addresses specific to network type.
```

Comment:
What is meant by "Unix domain name"? Should the "Unix" be removed here?

Reply:
oh, this was supposed to be "Unix domain socket names" (if my understanding is correct), fixed